Engine/Module/ModuleRender.cpp

```diff
@@ -195,6 +199,7 @@ void ModuleRender::RenderFrame(const ComponentCamera &camera)
 			mesh.second->Render();
 			num_rendered_tris += mesh.second->mesh_to_render->GetNumTriangles();
 			num_rendered_verts += mesh.second->mesh_to_render->GetNumVerts();
+			App->lights->UpdateLightAABB(*mesh.second->owner);
 			glUseProgram(0);
 
```

Comment:
While rendering directional light depth map the line `App->lights->UpdateLightAABB(*mesh.second->owner);` is called constantly, modyfing the light AABB when it shouldn't